lib/embed/embed.dart

```diff
@@ -337,12 +337,43 @@ class PlaygroundMobile {
     }
   }
 
+  bool validFlex(String input) {
+    return input != null && double.parse(input) > 0.0 && double.parse(input) < 1.0;
+  }
+  
+  int roundFlex(double flex) {
+    return (flex*10.0).round();
+  }
+  
   void _initPlayground() {
     // Set up the iframe.execution
     registerExecutionService();
     executionService.onStdout.listen(_showOutput);
     executionService.onStderr.listen((m) => _showOutput(m, error: true));
-
+    
+    //Set up the splitters
+    Uri url = Uri.parse(window.location.toString());
+    String v = url.queryParameters['verticalRatio'];
+    String h = url.queryParameters['horizontalRatio'];
+    left() => $('#left');
```

Comment:
Do you mean to be defining closures here? This would work just as well with `Element leftPanel = $('#leftPanel');`.